doc/quickstart/dashboard.md

```diff
@@ -0,0 +1,30 @@
+# Deploy a dashboard
+
+## 1. Click on "Dashboards" -> "New"
+
+![](../static/dashboards-new.png)
+
+
+## 2. Fill out the form
+
+
+![](../static/dashboards-form.png)
+
+## 3. Wait for the deployment to finish
+
+First, you'll see the "Deployment logs", which show the progress of installing your dashboards's dependencies.
+
+After a minute or so, you'll see the "Application logs", which expose your dashboard as a web application. The first deployment takes about 2 minutes.
+
+![](../static/dashboards-logs.png)
+
+Once the deployment is ready you'll see this in the "Application logs":
+
+
+```
+Deployment with ID : {some id} completed successfully.
+```
+
+Scroll up and click on "View Dashboard"
+
+![](../static/dashboards-view.png)
```

Comment:
Can this be a bigger screenshot to understand the placement of the button better?